api/v1alpha1/tunnelbinding_types.go

```diff
@@ -23,12 +23,12 @@ import (
 // TunnelBindingSubject defines the subject TunnelBinding connects to the Tunnel
 type TunnelBindingSubject struct {
 	// Kind can be Service
-	//+kubebuilder:validation:Optional
-	//+kubebuilder:default:="Service"
+	// +kubebuilder:validation:Optional
```

Comment:
it looks like the formatter you're using isn't respecting the lack of space for machine-read instructions

Reply:
It's fine; newer versions of the Operator SDK automatically add spaces, as some linters require this convention.